aten/src/ATen/cuda/CublasHandlePool.cpp

```diff
@@ -47,6 +87,15 @@ cublasHandle_t getCurrentCUDABlasHandle() {
   auto handle = myPoolWindow->reserve(device);
   auto stream = c10::cuda::getCurrentCUDAStream();
   TORCH_CUDABLAS_CHECK(cublasSetStream(handle, stream));
+#ifndef USE_ROCM
+  cudaStream_t _stream = stream;
```

Comment:
why are you casting CUDAStream to cudaStream_t?

Reply:
IIRC, `cudaStream_t` is a pointer type that should be unique for each stream, and the intent is to use this pointer as part of the key for the current (cuBLAS handle + stream) -> workspace map.